openfe/setup/molecule.py

```diff
@@ -14,10 +14,14 @@ class Molecule:
     ----------
     rdkit : rdkit.Mol
         rdkit representation of the molecule
+    name : str, optional
+        if multiple Molecules with identical SMILES but differing positions
+        are used, a name must be given to differentiate these.  This name
+        will be used in the hash.
     """
-    def __init__(self, rdkit: RDKitMol):
+    def __init__(self, rdkit: RDKitMol, name: str = None):
```

Comment:
Not sure about typing here -- is `None` allowed implicitly as `str`? Any reason not to default to `""` anyway?

Reply:
I thought it'd be like that case where a empty list is mutable, but it's not